Formula/khard.rb

```diff
@@ -3,9 +3,9 @@ class Khard < Formula
 
   desc "Console carddav client"
   homepage "https://github.com/scheibler/khard/"
-  url "https://files.pythonhosted.org/packages/5e/55/a4b34562af0f604b703fde449e18d4b452205daa303aeff832b6387f8849/khard-0.16.1.tar.gz"
-  sha256 "0d9f67a575ac945365c7748cf71f39d3b2281462977254d2e325ceff0ac4e439"
-  license "GPL-3.0"
+  url "https://files.pythonhosted.org/packages/a3/4e/e9cbcb281d371c355f251e5d9ca58b7e0d02dffd2bf4938888068fbc2def/khard-0.17.0.tar.gz"
+  sha256 "164e1aee9264735ec0473a74a38842e6272bbb814d949a66084c6a373bd95618"
+  license "GPL-3.0-or-later"
```

Comment:
Where do you see the `-or-later`? I don't see any indicator either way which indicates `-only`.

Reply:
I saw in the license file (bundled with the artifact) with the license header indicating this covers the later.

Reply:
Sorry, which file are you looking at? I don't see anything that indicates "or later" either.

Reply:
`LICENSE` file in the artifact, here is my reference:

```
  To do so, attach the following notices to the program.  It is safest
to attach them to the start of each source file to most effectively
state the exclusion of warranty; and each file should have at least
the "copyright" line and a pointer to where the full notice is found.

    <one line to give the program's name and a brief idea of what it does.>
    Copyright (C) <year>  <name of author>

    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

    You should have received a copy of the GNU General Public License
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Reply:
yeah, I think that is probably not quite sufficient point while I am re-reading the spdx license bodies.

Reply:
yeah, you guys are right, there is whatsoever license indication in the artifact to differentiate the `-only` or `-or-later`, do you want me to exclude the license update and move the change forward as it is, we can totally discuss this improvement in a separate PR.

Reply:
@lucc, @scheibler, not sure if you folks can help out with the license usage in here, as `GPL-3.0` is a deprecated license. Much appreciated.

Reply:
Sorry I can not help with that. The licence was originally chosen by @scheibler.